{
  "done": [
    "geotiff",
    "libspatialite",
    "gdal"
  ],
  "in-pr": [
    "vtk",
    "pyproj",
    "rasterio",
    "magics",
    "r-proj4"
  ],
  "awaiting-pr": [
    "magics-metview",
    "r-gdalcubes",
    "pynio",
    "pygplates",
    "postgis",
    "ncl",
    "mapserver",
    "fimex",
    "magics-metview-batch",
    "paraview"
  ],
  "not-solvable": [
    "r-terra",
    "r-sf",
    "r-rgdal"
  ],
  "awaiting-parents": [
    "visan",
    "r-lwgeom",
    "qgis",
    "geowombat",
    "cdo",
    "r-gribr"
  ],
  "bot-error": [],
  "_feedstock_status": {
    "cdo": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": ""
    },
    "gdal": {
      "num_descendants": 13,
      "immediate_children": [
        "geowombat",
        "mapserver",
        "ncl",
        "postgis",
        "pygplates",
        "pynio",
        "qgis",
        "r-gdalcubes",
        "r-rgdal",
        "r-sf",
        "r-terra",
        "rasterio"
      ],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/gdal-feedstock/pull/744",
      "pr_status": "unknown"
    },
    "geotiff": {
      "num_descendants": 14,
      "immediate_children": [
        "gdal",
        "qgis"
      ],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/geotiff-feedstock/pull/54",
      "pr_status": "unknown"
    },
    "libspatialite": {
      "num_descendants": 14,
      "immediate_children": [
        "gdal",
        "qgis"
      ],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/libspatialite-feedstock/pull/93",
      "pr_status": "unknown"
    },
    "magics": {
      "num_descendants": 1,
      "immediate_children": [
        "cdo"
      ],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/magics-feedstock/pull/121",
      "pr_status": "unstable"
    },
    "ncl": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": ""
    },
    "paraview": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": ""
    },
    "postgis": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": ""
    },
    "pynio": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": ""
    },
    "pyproj": {
      "num_descendants": 2,
      "immediate_children": [
        "geowombat",
        "qgis"
      ],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/pyproj-feedstock/pull/132",
      "pr_status": "unstable"
    },
    "qgis": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": ""
    },
    "r-proj4": {
      "num_descendants": 1,
      "immediate_children": [
        "r-gribr"
      ],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/r-proj4-feedstock/pull/40",
      "pr_status": "unstable"
    },
    "r-rgdal": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "not solvable (<a href=\"https://github.com/regro/cf-scripts/actions/runs/4462468126\">bot CI job</a>): main: ['linux_64_r_base4.1: Encountered problems while solving:\\n  - package r-base-4.1.3-h2f963a2_5 requires jpeg >=9e,<10a, but none of the providers can be installed\\n', 'osx_64_r_base4.1: Encountered problems while solving:\\n  - package r-base-4.1.3-h490d416_4 requires jpeg >=9e,<10a, but none of the providers can be installed\\n']"
    },
    "r-sf": {
      "num_descendants": 1,
      "immediate_children": [
        "r-lwgeom"
      ],
      "pre_pr_migrator_status": "not solvable (<a href=\"https://github.com/regro/cf-scripts/actions/runs/4462468126\">bot CI job</a>): main: ['linux_64_r_base4.1: Encountered problems while solving:\\n  - package r-base-4.1.3-h2f963a2_5 requires jpeg >=9e,<10a, but none of the providers can be installed\\n', 'linux_aarch64_r_base4.1: Encountered problems while solving:\\n  - package r-base-4.1.3-ha74fc48_5 requires jpeg >=9e,<10a, but none of the providers can be installed\\n', 'linux_ppc64le_r_base4.1: Encountered problems while solving:\\n  - package r-base-4.1.3-h0339487_4 requires jpeg >=9e,<10a, but none of the providers can be installed\\n', 'osx_64_r_base4.1: Encountered problems while solving:\\n  - package r-base-4.1.3-h490d416_4 requires jpeg >=9e,<10a, but none of the providers can be installed\\n']"
    },
    "rasterio": {
      "num_descendants": 1,
      "immediate_children": [
        "geowombat"
      ],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/rasterio-feedstock/pull/264",
      "pr_status": "unstable"
    },
    "vtk": {
      "num_descendants": 1,
      "immediate_children": [
        "visan"
      ],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/vtk-feedstock/pull/292",
      "pr_status": "clean"
    },
    "r-gribr": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": ""
    },
    "r-lwgeom": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": ""
    },
    "magics-metview": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": ""
    },
    "magics-metview-batch": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": ""
    },
    "geowombat": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": ""
    },
    "fimex": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": ""
    },
    "r-gdalcubes": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": ""
    },
    "r-terra": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "not solvable (<a href=\"https://github.com/regro/cf-scripts/actions/runs/4462468126\">bot CI job</a>): main: ['linux_64_r_base4.1: Encountered problems while solving:\\n  - package libgdal-3.6.3-habbca78_2 requires geos >=3.11.2,<3.11.3.0a0, but none of the providers can be installed\\n', 'linux_64_r_base4.2: Encountered problems while solving:\\n  - package libgdal-3.6.3-habbca78_2 requires geos >=3.11.2,<3.11.3.0a0, but none of the providers can be installed\\n', 'linux_aarch64_r_base4.1: Encountered problems while solving:\\n  - package libgdal-3.6.3-hb382bf2_2 requires geos >=3.11.2,<3.11.3.0a0, but none of the providers can be installed\\n', 'linux_aarch64_r_base4.2: Encountered problems while solving:\\n  - package libgdal-3.6.3-hb382bf2_2 requires geos >=3.11.2,<3.11.3.0a0, but none of the providers can be installed\\n', 'linux_ppc64le_r_base4.1: Encountered problems while solving:\\n  - package libgdal-3.6.3-h37cbc60_2 requires geos >=3.11.2,<3.11.3.0a0, but none of the providers can be installed\\n', 'linux_ppc64le_r_base4.2: Encountered problems while solving:\\n  - package libgdal-3.6.3-h37cbc60_2 requires geos >=3.11.2,<3.11.3.0a0, but none of the providers can be installed\\n', 'osx_64_r_base4.1: Encountered problems while solving:\\n  - package libgdal-3.6.3-h500a207_2 requires geos >=3.11.2,<3.11.3.0a0, but none of the providers can be installed\\n', 'osx_64_r_base4.2: Encountered problems while solving:\\n  - package libgdal-3.6.3-h500a207_2 requires geos >=3.11.2,<3.11.3.0a0, but none of the providers can be installed\\n']"
    },
    "visan": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": ""
    },
    "mapserver": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": ""
    },
    "pygplates": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": ""
    }
  }
}